catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -555,8 +558,12 @@ public Table createTable(
       Map<String, String> properties,
       Transform[] partitioning,
       Distribution distribution,
-      SortOrder[] sortOrders)
+      SortOrder[] sortOrders,
+      Index[] indexes)
       throws NoSuchSchemaException, TableAlreadyExistsException {
+    Preconditions.checkArgument(
```

Comment:
It's odd for me to check `indexes == Indexes.EMPTY_INDEXES`,  should we check partition like`partitions == Partitions.EMPTY`?  I'm clear to check indexes.isEmpty

Reply:
I think using `Indexes.EMPTY_INDEXES` is more readable because it clearly indicates that we are referring to an empty index. This is also the reason why `Indexes.EMPTY_INDEXES` was defined.

Reply:
Or the `Indexes.EMPTY_INDEXES` should be renamed to `Indexes.NONE`

Reply:
@FANNG1 @jerryshao cc

Reply:
I prefer to use `indexes.isEmpty` because it's a common way and the developers no need to know extra knowledge about `Indexes.EMPTY_INDEXES` or `Indexes.NONE`

Reply:
`indexes.isEmpty` also good for me